paket.dependencies

```diff
@@ -4,6 +4,7 @@ storage: none
 framework: net6.0
 
 nuget Expecto
+nuget FSharp.Control.Websockets 0.3.0
```

Comment:
what impact does this have on the nuspec for Saturn? I want to make sure that this doesn't pin users to 0.3.0 and instead a constraint of `[0.3.0, )` is created.

Reply:
I'm not sure, but I think it's safe to remove this specific number. Let me try it here.

Reply:
I updated the code to use the same `>=` operator as before. Tested locally and it's still working fine.